.claude/skills/twelve-factor-methodology/scripts/bus.sh

```
@@ -0,0 +1,318 @@
+
+# ============================================================================
+# SELF-TEST
+# Usage: bus_selftest
+# ============================================================================
+bus_selftest() {
+  echo "=== Bus Self-Test ==="
+  echo "Backend: $BACKEND"
+  echo "Events: $EVENTS_PATH"
+
+  # Test publish
+  publish_event "test.ping" '{"msg":"hello","ts":"'$(ts_now)'"}'
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)